docs/releases/5.1.txt

```diff
@@ -227,6 +227,13 @@ Tests
 * The Django test runner now supports a ``--screenshots`` option to save
   screenshots for Selenium tests.
 
+* The Django test runner now supports a ``--pair`` option that performs a pairwise
```

Comment:
Please rephrase this as per https://github.com/django/django/pull/17799 (pending its merge).